Wrapping/Generators/Python/itk/support/helpers.py

```diff
@@ -173,3 +176,117 @@ def image_filter_wrapper(*args, **kwargs):
             return image_filter(*args, **kwargs)
 
     return image_filter_wrapper
+
+
+def image_to_type(itkimage):  # noqa: C901
```

Comment:
To be more specific and consistent, we could call this `wasm_type_from_image` and return

```
    imageType = dict(
      dimension=dimension,
     componentType=componentType,
     pixelType=pixelType,
     components=itkimage.GetNumberOfComponentsPerPixel()
   )
  return imageType
```

Reply:
`dimension` and `componentType` are undefined in that function, are they named something else there instead?

I maaaybe guess:
- ``componentType` == `_python_to_js[mangle]`
- `dimension` == `pixelType`

...but it's late here, so I'm going to stop for today.

Reply:
Wait no, I think that's wrong.

...I'll just change the function name, for now. We can talk about this bit more later.

Reply:
`componentType==_python_to_js[mangle]`

Correct

```
dimension = itkimage.GetImageDimension()
components = itkimage.GetNumberOfComponentsPerPixel()
```